Resources/Locale/en-US/_Goobstation/changeling/store/changeling-catalog.ftl

```diff
@@ -1,140 +1,125 @@
-
-# combat
+# Combat
 
 evolutionmenu-combat-armblade-name = Arm Blade
 evolutionmenu-combat-armblade-desc =
-    Reform one of your arms into a grotesque blade, composed of bone and flesh, able to pry open airlocks and cut through your foes like butter.
-    Costs 15 chemicals.
+    We reform one of our arms into a deadly blade.
+    Costs 20 chemicals.
 
 evolutionmenu-combat-boneshard-name = Bone Shard
 evolutionmenu-combat-boneshard-desc =
-    Break off shards of your bone and shape them into a throwing star which embeds into your foes. But a one timer opportinuty.
+    We break off shards of sharp bone to use as projectile weapons.
     Costs 15 chemicals.
 
 evolutionmenu-combat-armor-name = Chitinous Armor
 evolutionmenu-combat-armor-desc =
-    Inflate your body into an all-consuming chitinous mass of armor.
-    Provides extensive protection against physical damage, but less against other types.
-    It massively slows your movement, and maintaining its shape slows chemical generation.
-    WARNING: Requires you to absorb at least 2 organics to use the ability.
+    We turn our skin into tough chitin to protect us from damage. Upkeep of the armor requires a low expenditure of chemicals.
+    WARNING: We must absorb at least 2 organics to use this ability.
     Costs 25 chemicals.
 
 evolutionmenu-combat-shield-name = Organic Shield
 evolutionmenu-combat-shield-desc =
-    Reforms one of your arms into a large, fleshy shield.
-    Blocks attacks automatically, but very brittle.
+    We reform one of our arms into a hard, yet brittle shield.
     WARNING: Requires you to absorb at least 1 organic to use the ability.
     Costs 20 chemicals.
 
 evolutionmenu-combat-shriek-dissonant-name = Dissonant Shriek
 evolutionmenu-combat-shriek-dissonant-desc =
-    You emit an EMP blast, which disables technology in the surrounding area, including radio headsets.
-    Good for escaping cyborgs and security.
+    We shift our vocal cords to release a high-frequency sound that overloads nearby electronics.
     WARNING: Requires you to absorb at least 1 organic to use the ability.
-    Costs 30 chemicals.
+    Costs 20 chemicals.
 
 evolutionmenu-combat-shriek-resonant-name = Resonant Shriek
 evolutionmenu-combat-shriek-resonant-desc =
-    You emit a tone beyond the range of human hearing,
-    bursting lights and causing disorientation in an area around yourself.
-    Good for escaping groups, or hindering people from fleeing.
+    Our lungs and vocal cords shift, allowing us to briefly emit a noise that defans and confuses the weak-minded.
```

Comment:
im trying to de-bulk a lot of these longer descriptions since theyre all so fucking huge in the evolution point store. maybe if changelings had their own cool store instead of just snagging traitor uplinks..

Reply:
changed to `Our lungs and vocal cords shift, allowing us to briefly emit a noise that deafens and confuses our prey.` (includes a typo fix whoopsie!) but id really like to avoid bulking up the descriptions with "good for" text. i can relent if this is a big sticking point, though

Reply:
is defan an alternate spelling of deafen i have never seen before. i am getting vampire diaries ships when i check so im assuming this is a typo